operator-metadata/manifests/strimziclusteroperator.configmap.yaml

```diff
@@ -1,7 +1,7 @@
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: amq-streams-cluster-operator
+  name: strimzi-cluster-operator
```

Comment:
I think you should keepo the original name and change only the content. Or alternatively, you need to change the mount in the operator deployment:
```yaml
                - name: co-config-volume
                  configMap:
                    name: amq-streams-cluster-operator
```

Reply:
right